Jenkinsfile-Release

```diff
@@ -90,8 +90,10 @@ dockerizedBuildPipeline(
     String version = verifyIndexYamlAndTarFile(params.chart)
 
     // Set Jira Fix Version
-    jira.versionPrefix = jiraVersionMappings[chart]
-    jiraSetFixVersion(jira, version)
+    if ('nexus-iq-server'.equals(params.chart)) {
```

Comment:
nit: this is groovy, so technically, you can safely use the '=' operator in place of `equals` method. (i missed this in a previous PR.)